### ISSUE TITLE
rds.RDSClient.generate_db_auth_token() lacks proper return value

### ISSUE THREAD

**Issue by hakamadare** [vemel/mypy_boto3_builder#12]
**Describe the bug**
`rds.RDSClient.generate_db_auth_token()`'s type signature returns `None`, when the actual function returns `str`

**To Reproduce**
1. Install 'boto3-stubs[rds]'
2.  Run mypy on the following code sample

```python
import boto3
from mypy_boto3 import rds

r: rds.RDSClient = boto3.client("rds")

auth_token = r.generate_db_auth_token("host", 5432, "user")
```
3. Actual mypy output

```
$ mypy test.py
test.py:6: error: "generate_db_auth_token" of "RDSClient" does not return a value
Found 1 error in 1 file (checked 1 source file)
```

**Expected mypy-output**
```
$ mypy test.py
Success: no issues found in 1 source file
```

**Additional context**
```
$ pip freeze | grep boto
boto==2.49.0
boto3==1.12.28
boto3-stubs==1.12.28.0
botocore==1.15.28
mypy-boto3==1.12.28.0
mypy-boto3-rds==1.12.28.0
```
this is on macOS 10.15.3, running in a virtualenv.

per [boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Client.generate_db_auth_token) this function is supposed to return a signed URL.

this is the generated type definition:
```
def generate_db_auth_token(
        self, DBHostname: str, Port: int, DBUsername: str, Region: str = None
    ) -> None:
        """
        [Client.generate_db_auth_token documentation](https://boto3.amazonaws.com/v1/documentation/api/1.12.28/reference/services/rds.html#RDS.Client.generate_db_auth_token)
        """
```

**Comment by vemel** [comment]
This method's type annotations are generated from a docstring.  I will check, thanks.

**Comment by vemel** [comment]
Indeed, no return type in docstring:

```
Generates an auth token used to connect to a db with IAM credentials.

:type DBHostname: str
:param DBHostname: The hostname of the database to connect to.

:type Port: int
:param Port: The port number the database is listening on.

:type DBUsername: str
:param DBUsername: The username to log in as.

:type Region: str
:param Region: The region the database is in. If None, the client
    region will be used.

:return: A presigned url which can be used as an auth token.
```

I can detect `presigned url` in `return` section. fi it present - i set return type to `str`

**Comment by hakamadare** [comment]
@vemel just for my future reference, where did you look to find this docstring?  next time i'd like to be able to submit a patch :)

**Comment by vemel** [comment]
I do it using `__doc__`, but they are generated from botocore shapes.

```
import boto3

client = boto3.client("rds")
help(client.generate_db_auth_token)
```

Should be fixed in the latest boto3-stubs==1.12.28.1. I also recommend to update boto3 to 1.12.28 to keep type annotations consistent.

Please let me know if the issue still exists or if you find something related.

**Comment by hakamadare** [comment]
just confirmed fixed!  thank you for your rapid response, well done :)
